website/content/docs/concepts/architecture.md

```diff
@@ -213,3 +213,53 @@ The source for this diagram lives here:
 https://docs.google.com/drawings/d/10d1Q_VO0HFtOHftDV7kK6VbZMVI5EpEYHrD-LR7SczE
 -->
 <img src="/img/topology-submit-sequence-diagram.png" style="max-width:140%;!important;" alt="Topology Sequence Diagram"/>
+
+### Brief Description
+
+The following briefly describes how a topology is handled internally.
+
+* Client
+
+    When a topology is submitted using `heron submit` command, it first executes
+    the `main` of the topology and creates `.defn` file containing the topology's
+    logical plan. Then, it runs `com.twitter.heron.scheduler.SubmitterMain`, which
+    is responsible for invoking an uploader and a launcher for the topology.
+    The uploader uploads the topology package to the given location, while the
+    launcher registers the topology's logical plan and executor state with 
+    the state manager and invokes the main scheduler.
+
+* Shared Services
+
+    When the main scheduler (`com.twitter.heron.scheduler.SchedulerMain`) is invoked 
```

Comment:
@billonahill Do you think it would be better to provide an Aurora specific description as well, or just to note that this flow is an example of the local scheduler?